go.mod

```diff
@@ -6,6 +6,6 @@ require (
 	github.com/pion/logging v0.2.2
 	github.com/pion/rtcp v1.2.9
 	github.com/pion/rtp v1.7.13
-	github.com/pion/transport v0.13.0
+	github.com/pion/transport v0.13.1-0.20220609023836-26aa3f6dca85
```

Comment:
I'm not sure if this is desirable. Do we need to wait for @Sean-Der to 'prod' pion/transport to v0.13.1 so we're referencing a release instead of a dated commit?

Reply:
Yes, I think so.

Reply:
I've now pushed the v0.13.1 tag to pion/transport. Can you try fixing up go.mod accordingly and we'll see if that works?